tests/unit/src/presenters/question-to-inset.test.js

```diff
@@ -4,42 +4,82 @@ const moment = require("moment");
 describe("question-to-inset", () => {
   let translate;
   let question;
-  let locals;
+  let data;
   const language = "en";
 
   beforeEach(() => {
     question = {
       questionKey: "rti-payslip-national-insurance",
+      info: {
+        months: 3,
```

Comment:
In `questions.json` which mocks up what we'll get from the API, the months value is a string